Sources/Purchasing/StoreKit2/SK2AppTransaction.swift

```diff
@@ -25,6 +24,18 @@ internal struct SK2AppTransaction {
         self.environment = .init(environment: appTransaction.environment)
     }
 
+    init(
```

Comment:
This is only used for tests, right?

In that case, I'd move it to an extension in tests only so we don't ship this, since it's only for creating mock transactions.